test/e2e/Tests/E2ETests.csproj

```diff
@@ -5,7 +5,8 @@
     <LangVersion>latest</LangVersion>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <RunSettingsFilePath>$(MSBuildProjectDirectory)\test.runsettings</RunSettingsFilePath>
+    <!-- Only add the following line when you want to locally debug
+    <RunSettingsFilePath>$(MSBuildProjectDirectory)\test.runsettings</RunSettingsFilePath> -->   
```

Comment:
This has to be commented out. Otherwise, environment variables set in the E2ETest.yml will be overwritten by this file and thus all the E2E CI will use AzureStorage backend

Reply:
Good catch! Looks like there are some mssql tests failing now, we'll have to address those, but it can be done separately